client/src/components/Ticketing/ticketingmanager/Doorlistpage/doorlist.tsx

```diff
@@ -82,86 +86,189 @@ const DoorList = () => {
   const [eventName, setEventName] = useState('');
   const [date, setDate] = useState('');
   const [time, setTime] = useState('');
-
+  const [selectedEventId, setSelectedEventId] = useState(null);
+  const [availableTimesEvents, setAvailableTimesEvents] = useState([]);
   const [eventList, setEventList] = useState([]);
-
-  const getEvents = async () => {
-    try {
-      const response = await fetch(process.env.REACT_APP_API_1_URL + '/events/list/active');
-      const jsonRes = await response.json();
-      const jsonData = jsonRes.data;
-      Object.keys(jsonData).forEach(function(key) {
-        jsonData[key].eventdate = dayMonthDate(jsonData[key].eventdate);
-        jsonData[key].eventtime = militaryToCivilian(jsonData[key].eventtime);
-      });
-      setEventList(jsonData);
-    } catch (error) {
-      console.error(error.message);
-    }
-  };
+  const [eventListFull, setEventListFull] = useState([]);
+  const [eventListActive, setEventListActive] = useState([]);
+  const [ticketsSold, setTicketsSold] = useState(true);
+  const [showInactiveEvents, setShowInactiveEvents] = useState(false);
 
   useEffect(() => {
-    getEvents();
+    const fetchEvents = async () => {
+      try {
+        const response = await fetch(process.env.REACT_APP_API_1_URL + '/events/list/allevents');
```

Comment:
This looks good! I would recommend adding a check right after this fetch call to check if response.ok is true, then proceed, else throw an error. The reason is because if the response HTTP status code is 400 or 500 level which indicates error codes, it technically is a "success" and the try block will continue executing even though we don't want too.